### PR TITLE
Update aws-privatelink.md

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/aws-privatelink.md [hunk 1]
```diff
@@ -13,13 +13,26 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 <p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
 
-[AWS PrivateLink](https://aws.amazon.com/privatelink) provides private connectivity between VPCs, AWS services, and your on-premises networks, without exposing your traffic to the public internet.
+Sumo Logic provides the ability to configure private connectivity between your AWS Infrastructure and Sumologic via [AWS PrivateLink](https://aws.amazon.com/privatelink). This prevents any traffic from being accessible to the public internet. 
 
-To send data to Sumo Logic through AWS PrivateLink, you'll need to configure an internal endpoint in your VPC for Installed Collectors to send data to.
+Following sources are supported for AWS PrivateLink: 
+
+- **Installed Collector sources**. Data collected on Installed collectors deployed on customer VPC and sent to the Sumo Logic.
+- **HTTPs sources**. Data sent by the applications hosted in customers VPC and sent over to Sumologic using [HTTP Logs and Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/) and Kinesis Firehose Source for Logs and Metrics.
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/aws-privatelink.md:21]:
```suggestion
- **HTTPs sources**. Data sent by the applications hosted in customers VPC and sent over to Sumologic using [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics/) and Kinesis Firehose Source for Logs and Metrics.
```

File: docs/integrations/amazon-aws/aws-privatelink.md [hunk 2]
```diff
@@ -13,13 +13,26 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+Following sources are not supported for AWS PrivateLink:
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/aws-privatelink.md:23]:
```suggestion
The following sources are not supported for AWS PrivateLink: 
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I have a couple of suggestions for minor updates.

File: docs/integrations/amazon-aws/aws-privatelink.md (hunk 1 above)
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-privatelink.md:21]:
```suggestion
- **HTTPS sources**. Data sent by the applications hosted in customers VPC and sent over to Sumo Logic using [HTTP Logs and Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/) and Kinesis Firehose Source for Logs and Metrics.
```

**Pull request merged**